### PR TITLE
sccp 215/212/214/195

### PR DESCRIPTION
**Pull request by kaleb-keny**

### CODE REVIEW

**Comment by kaleb-keny** [comment]
Hey @artdgn can you please have a look, this implements sccp-215/212/214/195 which are currently being voted on and are expected to pass

**Comment by kaleb-keny** [comment]
https://gov.synthetix.io/#/

File: publish/deployed/mainnet-ovm/owner-actions.json [hunk 1]
```diff
@@ -1 +1,9 @@
-{}
+{
+    "0xaE55F163337A2A46733AA66dA9F35299f9A46e9e.setMinKeeperFee(uint256):1000000000000000000": {
```
[the remark this answers is not in the document]
Comment (kaleb-keny) [reply publish/deployed/mainnet-ovm/owner-actions.json]: Done sir, thank you did not know about this. When you say `The effects will be reviewed on the safe` what do you mean?

[the remark this answers is not in the document]
Comment (artdgn) [reply publish/deployed/mainnet-ovm/owner-actions.json]: That files hosts the owner actions that will be published to the owner gnosis safe, and the tx will be reviewed (by the multisig owners) to correspond to the SCCPs. So the changes to this file in this repo aren't needed.

File: index.js [hunk 2]
```diff
@@ -208,7 +208,7 @@ const defaults = {
 	ETHER_WRAPPER_MINT_FEE_RATE: w3utils.toWei('0.005'), // 5 bps
 	ETHER_WRAPPER_BURN_FEE_RATE: '0',
 
-	FUTURES_MIN_KEEPER_FEE: w3utils.toWei('5'), // 5 sUSD min keeper fee
+	FUTURES_MIN_KEEPER_FEE: w3utils.toWei('1'), // 5 sUSD min keeper fee
```
[the remark this answers is not in the document]
Comment (kaleb-keny) [reply index.js]: thank you, fixed it

**Comment by kaleb-keny** [comment]
hey @artdgn this seems to be giving errors, related to fee recl being set to zero on optimism and wrappr being at capacity. Anything we can do here to merge this?

**Pull request merged**